src/components/decimal/decimal.js

```diff
@@ -111,14 +107,23 @@ class Decimal extends React.Component {
     precision: 2
   };
 
+  // Caps precision at 20 to prevent browser issues
+  verifiedPrecision = (precision) => {
+    if (precision > 20) {
+      console.warn('Decimal precision must not be greater than 20.'); // eslint-disable-line no-console
```

Comment:
should use our logger for this, then you don't need the eslint disable and it won't get output in production mode